var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -132,8 +132,8 @@ def configure_args(self):
             options.append("--enable-generic")
 
         if self.compiler.name == "gcc":
-            gcc_options = []
-            gfortran_options = ["-ffree-line-length-none"]
+            gcc_options = ["-O3"]
+            gfortran_options = ["-O3", "-ffree-line-length-none"]
```

Comment:
Feel free to simplify to `options.extend(["CFLAGS=-O3", "FFLAGS=-O3 -ffree-line-length-none"])` cause I don't see the point of `space_separator` :smile: